pode.build.ps1

```diff
@@ -279,14 +279,24 @@ task Test Build, TestDeps, {
     }
 
     $Script:TestResultFile = "$($pwd)/TestResults.xml"
+    # get default from static property
+    $configuration = [PesterConfiguration]::Default
+    $configuration.run.path= @('./tests/unit', './tests/integration')
+    $configuration.run.PassThru=$true
+    $configuration.CodeCoverage.OutputFormat = 'NUnitXml'
 
+    $configuration.TestResult.OutputPath = $Script:TestResultFile
     # if run code coverage if enabled
     if (Test-PodeBuildCanCodeCoverage) {
         $srcFiles = (Get-ChildItem "$($pwd)/src/*.ps1" -Recurse -Force).FullName
-        $Script:TestStatus = Invoke-Pester './tests/unit', './tests/integration' -OutputFormat NUnitXml -OutputFile $TestResultFile -CodeCoverage $srcFiles -PassThru
-    }
-    else {
-        $Script:TestStatus = Invoke-Pester './tests/unit', './tests/integration' -OutputFormat NUnitXml -OutputFile $TestResultFile -Show Failed -PassThru
+        $configuration.CodeCoverage.Enabled = $true
+
+        #  $Script:TestStatus = Invoke-Pester -Path './tests/unit', './tests/integration'  -OutputFormat NUnitXml -OutputFile $TestResultFile -CodeCoverage $srcFiles -PassThru
+        $Script:TestStatus = Invoke-Pester   -Configuration $configuration
+    } else {
+        $configuration.Output.Verbosity= 'Detailed'
```

Comment:
To reflect the original this should be `Normal` instead of `Detailed`; if we want to allow the verbosity to be toggled I'd recommend adding a new parameter at the top of this file :)

Reply:
done